cmd/util/common.go

```diff
@@ -104,6 +104,7 @@ type Config struct {
 	// implementations.
 	Bugs CertificateBugs
 
+	// The signature algorithm for the Delegated Credential
```

Comment:
Doc strings for parameters being with the name of the parameter.
```
// SignatureAlgorithm ....
```

Also, since this variable is exported, I'm wondering if we should export its type as well?

Reply:
Ah.. this wasn't the comment ;) I putted it there to remember ;) But I can add the comments as well.

Reply:
Ha no worries. I guess I should hold off on these kinds of comments until it's no longer "WIP" and is "ready for review"?

Reply:
Solved now. Open for review as well.